interfaces/tests/lock_ut/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(theseTestsName lock_ut)
```

Comment:
actually the lock unit tests belong in the win32 folder instead

Reply:
Because technically these should also mock the win32/linux parts. For now you could copy this into both and then we could do better mocking later